src/client/pfs/pfs.proto

```diff
@@ -312,12 +312,20 @@ message TagObjectRequest {
   repeated Tag tags = 2;
 }
 
+message ListObjectsRequest {}
+
+message ListObjectsTaggedWithPrefixRequest {
+  string prefix = 1;
+}
```

Comment:
That seems fine to me, the `InspectTag` method below returns an `ObjectInfo` it's sort of in keeping with the idea that `Tag`s are just aliases for `Object`s